packages/yoroi-ergo-connector/src/inject.js

```diff
@@ -108,6 +126,25 @@ class ErgoAPI {
 
 const ergo = Object.freeze(new ErgoAPI());
 `
+
+const cardanoApiInject = `
+
+// @todo add cardano rpc resolver when it is done
+
+class CardanoAPI {
+    constructor(){
+        this.initTimestamp = Date.now()
+        this.getInitTimestamp()
```

Comment:
No reason to call this function in the constructor, it's just a getter with no side-effects

```suggestion
```